tests/unit/module_utils/test_role_utils.py

```diff
@@ -0,0 +1,145 @@
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2019, XLAB Steampunk <steampunk@xlab.si>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+from ansible_collections.sensu.sensu_go.plugins.module_utils import role_utils
+
+
+class TestSortRules:
+    def test_sort_rules(self):
+        assert role_utils._sort_rules(
+            [{'a': [2, 1]}, {'b': [4, 3]}]
+        ) == [{'a': [1, 2]}, {'b': [3, 4]}]
+
+    def test_sort_none_value(self):
+        assert role_utils._sort_rules(
+            [{'a': [2, 1]}, {'b': None}]
+        ) == [{'a': [1, 2]}, {'b': None}]
+
+
+class TestDoRulesDiffer:
+    def test_empty_values(self):
+        assert role_utils._do_rules_differ(
+            [{'a': []}],
+            [{'a': []}]
+        ) is False
+
+    def test_rules_when_value_is_none(self):
```

Comment:
I find this test a bit confusing, what exactly did you want to test here? Maybe it's just the actual test data that confuses me - I would at least make the value of `a` the same in current and desired rule set (or remove `a` entirely, if we want to test the behavior for `None` case in isolation)?

Reply:
Yeah, I wanted to test the None cases, so `a` should not really matter in that case.

Maybe it would've be better if I used the actuall keys and values used in real case data instead of `a` and `b` and numbers.